tests/xapian_tests/models.py

```diff
@@ -34,3 +34,26 @@ class BlogEntry(models.Model):
     number = models.IntegerField()
     float_number = models.FloatField()
     decimal_number = models.DecimalField(max_digits=4, decimal_places=2)
+
+
+class UUIDBlogEntry(models.Model):
```

Comment:
Would it be possible to minimize this model to focus on the relevant aspect, non-integer pk?